code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -74,9 +75,10 @@
 	var/A = get_turf(target)
 	to_chat(user, span_danger("You start drawing a rune..."))
 
-	if(do_after(user, 30 SECONDS, A))
-
-		new /obj/effect/eldritch/big(A)
+	if(do_after(user, 8 SECONDS, A))
```

Comment:
you should document that it's reduced to 8 seconds, not 10 seconds

Reply:
even BETTER. 30 seconds was ABSURD, 10 even feels like a lot.